### DISCUSSION TITLE
Abfrageinterval 3 Sek, aber Aktualisierung findet nur alle 15 Sek. statt!?

### DISCUSSION THREAD

**Discussion by fritzCARaldo** [tbnobody/OpenDTU#629] [Q&A]
Hallo, ich hoffe ich bin hier richtig:

Ich habe das **Abfrageintervall** auf meinem **OpenDTU** (Firmwareversion / git Hash | 97bc964 (aktuell)) **auf 3 Sek. eingestellt**, was in einer früheren Firmwareversion auch super geklappt hat. Sowohl mit der letzten, als auch der aktuellen FW (eben aktualisiert) findet die **Aktualisierung findet nur alle 15 Sek. statt**!?

Was mache ich falsch? Danke für eure Hilfe! David

**Comment by ghost** [comment]
Hast Du mehr als einen Inverter? Falls ja, gilt das Abfrageintervall für alle zusammen, so dass jeder einzelne entsprechend seltener an die Reihe kommt. Nur ein Gedanke...

**Reply by fritzCARaldo** [reply]
Ich hab zwar noch 2 HM-600 liegen, angeschlossen ist aber nur einer. Und das Intervall passte am Anfang auch. Seit dem vorletzten Update ist mir das aufgefallen.

**Comment by hakki1962** [comment]
Hallo zusammen bei mir die Aktualisierung Zeiten sind ganz unterschiedlich. von 3 bis 6000 sogar mehr Sekunden. DIE INTERVALZEIT WIE ORIGINAL 10s. woran kann es liegen?  Und habe nur ein Inverter HM-600.
